Makefile

```diff
@@ -148,13 +148,14 @@ vet: gowork ## Run go vet against code.
 	go vet ./...
 	go vet ./apis/...
 
+BRANCH=main
```

Comment:
so my plan is to change this on the next stable branch to the name of that branch. Then call this form renovate as post action to force dependencies to the stable version. 

On main this can still be useful if somebody wants to generate a bump commit outside of renovate but on main renovate already automates it (just in a different way)